functions/decrypt.php

```diff
@@ -0,0 +1,7 @@
+<?php
+/* 
+Author: AliceKarns 
+Version: 1.0
+Description: Decrypt.
+*/
+?>
```

Comment:
Only comments?

Reply:
@urbiswax Makesure to commits only codes that you've done.